crt_portal/cts_forms/forms.py

```diff
@@ -951,6 +965,49 @@ class Filters(ModelForm):
             'placeholder': 'yyyy-mm-dd',
         }),
     )
+    primary_complaint = MultipleChoiceField(
+        required=False,
+        choices=PRIMARY_COMPLAINT_PROFORM_CHOICES,
+        widget=UsaCheckboxSelectMultiple(attrs={
+            'name': 'primary_issue',
+        }),
+        label='Primary Issue',
+    )
+    reported_reason = MultipleChoiceField(
```

Comment:
Could be this be a `ModelMultipleChoiceField` like we're using for the `ProtectedClassForm`?

As-is this form generates unexpected escaping with:
1. Add a `reported_reason` search term
    * `race/color` results in URL param`reported_reason=race%2Fcolor` 
2. Modify another search term to search again
    * `reported_reason=race%2Fcolor` now becomes `reported_reason=race%252Fcolor`

Reply:
Huh, this is an interesting one. I don't know if we can use a `ModelMultipleChoiceField` here, it seems to make things worse. 

![2020-09-28_16-38](https://user-images.githubusercontent.com/3013175/94496399-1e09ed80-01a9-11eb-8fc0-707a6798421f.png)

I don't think it has anything necessarily to do with the form or widget, I think there's something else happening here. Investigating...

Reply:
If I'm remembering correctly I think we're using a shorthand of these for the details page, maybe we could use that data here instead of the public phrase?

Reply:
Yep, the bug here turned out to be in the javascript code itself: we were not URI decoding query params that came in (but we were certainly URI encoding params that went out). I think this is fixed, so resolving.